Update Configuration Variables section

tjorim opened the pull request:
**Description:**
- Updates the Configuration Variables section of several components according to the docs: https://developers.home-assistant.io/docs/en/documentation_create_page.html
- Fix some types to be in line with the rest
- Some other small typo's

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

tjorim committed:
Update Configuration Variables section

tjorim committed:
Remove leftover line

tjorim committed:
Update Configuration Variables section

tjorim committed:
Update camera.mjpeg.markdown

tjorim committed:
Update image_processing.microsoft_face_detect.markdown

tjorim committed:
Update image_processing.microsoft_face_detect.markdown

tjorim committed:
Update image_processing.microsoft_face_detect.markdown

tjorim committed:
Update image_processing.microsoft_face_detect.markdown

tjorim committed:
Update Configuration Variables section

tjorim committed:
Update image_processing.microsoft_face_identify.markdown

tjorim committed:
Update Configuration Variables section

tjorim committed:
Update image_processing.microsoft_face_detect.markdown

tjorim committed:
Update Configuration Variables section

tjorim committed:
Update image_processing.openalpr_cloud.markdown

tjorim committed:
Update Configuration Variables section

tjorim committed:
Update Configuration Variables section

tjorim committed:
Update Configuration Variables section

tjorim committed:
Update Configuration Variables section

tjorim committed:
Update image_processing.openalpr_local.markdown

tjorim committed:
Update notify.html5.markdown

tjorim committed:
Update notify.webostv.markdown

tjorim committed:
Fix typo

tjorim committed:
Newline

tjorim committed:
Update Configuration Variables section

tjorim committed:
Update Configuration Variables section

tjorim committed:
Update Configuration Variables section

tjorim committed:
Update Configuration Variables section

tjorim committed:
Update Configuration Variables section

tjorim committed:
Update Configuration Variables section

tjorim committed:
Update Configuration Variables section

tjorim committed:
Update image_processing.microsoft_face_detect.markdown

tjorim committed:
Update image_processing.openalpr_local.markdown

tjorim committed:
Update notify.html5.markdown

tjorim committed:
Update notify.webostv.markdown

tjorim committed:
Update Configuration Variables section

tjorim committed:
Update sensor.yr.markdown

tjorim committed:
webOS brand fix + filename clarification

tjorim committed:
Update Configuration Variables section

tjorim committed:
Update media_player.webostv.markdown

tjorim committed:
Update Configuration Variables section

tjorim committed:
Update media_player.samsungtv.markdown

tjorim committed:
Update Configuration Variables section

tjorim committed:
Update Configuration Variables section

tjorim committed:
Update Configuration Variables section

tjorim committed:
Update Configuration Variables section

tjorim committed:
Update Configuration Variables section

tjorim committed:
Update Configuration Variables section

tjorim committed:
Update Configuration Variables section

tjorim committed:
Update Configuration Variables section

tjorim committed:
Update Configuration Variables section

tjorim committed:
Update Configuration Variables section

tjorim committed:
Update Configuration Variables section

tjorim committed:
Update Configuration Variables section

tjorim committed:
Update Configuration Variables section

tjorim committed:
Update Configuration Variables section

tjorim committed:
Update Configuration Variables section

tjorim committed:
Fix capital letter

tjorim committed:
Update sensor.iota.markdown

tjorim committed:
Fix capital letter

tjorim committed:
Fix capital letter

tjorim committed:
Update http.markdown

tjorim committed:
Update media_player.webostv.markdown

tjorim committed:
Fix capital letter

tjorim committed:
Update notify.webostv.markdown

tjorim commented:
Would like to know the answer to https://github.com/home-assistant/home-assistant/pull/15143#issuecomment-408599718, to reflect correct options in the documentation.

dale3h commented on source/_components/binary_sensor.ihc.markdown:
```diff
@@ -52,8 +52,8 @@ binary_sensors:
-      type: bool
-      default: false
+      type: boolean
+      default: False
```
Please change to all lowercase (`default: false`).

dale3h commented on source/_components/cover.template.markdown:
```diff
@@ -87,13 +84,13 @@ cover:
-        type: bool
-        default: false
+        type: boolean
+        default: False
```
Please change to all lowercase (`default: false`).

dale3h commented on source/_components/cover.template.markdown:
```diff
@@ -87,13 +84,13 @@ cover:
-        type: bool
-        default: false
+        type: boolean
+        default: False
```
Please change to all lowercase (`default: false`).

dale3h commented on source/_components/cover.template.markdown:
```diff
@@ -102,36 +99,24 @@ cover:
-If you are using the state of a platform that takes extra time to load, the
-Template Cover may get an `unknown` state during startup. This results
-in error messages in your log file until that platform has completed loading.
+If you are using the state of a platform that takes extra time to load, the Template Cover may get an `unknown` state during startup. This results in error messages in your log file until that platform has completed loading.
```
Please revert back to <=80 characters per line.

dale3h commented on source/_components/cover.template.markdown:
```diff
@@ -102,36 +99,24 @@ cover:
 {% raw %}`{{ states.switch.source.state == 'on' }}`{% endraw %}
-with this equivalent that returns `true`/`false` and never gives an unknown
-result:
+with this equivalent that returns `true`/`false` and never gives an unknown result:
```
Please revert back to <=80 characters per line.

dale3h commented on source/_components/cover.template.markdown:
```diff
@@ -102,36 +99,24 @@ cover:
-attribute. There is an equivalent mode for `tilt_position` that is enabled
-when [`tilt_template`](#tilt_template) is not specified or when the
-[`tilt_optimistic`](#tilt_optimistic) attribute is used.
+In optimistic mode, the cover position state is maintained internally. This mode is automatically enabled if neither [`value_template`](#value_template) or [`position_template`](#position_template) are specified. Note that this is unlikely to be very reliable without some feedback mechanism, since there is otherwise no way to know if the cover is moving properly. The cover can be forced into opti ...
```
Please revert back to <=80 characters per line.

dale3h commented on source/_components/cover.template.markdown:
```diff
@@ -102,36 +99,24 @@ cover:
 
-This example converts a garage door with a controllable switch and position
-sensor into a cover.
+This example converts a garage door with a controllable switch and position sensor into a cover.
```
Please revert back to <=80 characters per line.

dale3h reviewed (changes requested):
- Please change all `default: True` to `default: true` (preferred for YAML)
- Please change all `default: False` to `default: false` (preferred for YAML)
- Please reduce all lines to 80 characters or less where possible. To reduce lines with URLs, it is preferred to use reference links in the entire document.

This is just a preliminary review. Another review will be conducted after these changes have been addressed.

tjorim committed:
Lint fixes etc.

All default True/False capital letters have been fixed.
Tried to reduce lines to 80 characters or less where possible.

tjorim committed:
Try multiline string in Cast

Trying out a multiline string in a {% configuration %}-block.

tjorim committed:
The rest for this PR

- Update components with new Configuration Variables section to have YAML block syntax
- Fix wrong capitalised booleans

tjorim committed:
Fix Liquid Exception sensor.speedtest

tjorim committed:
More updated Configuration Variables sections

tjorim committed:
Update binary_sensor.netatmo.markdown

tjorim committed:
Fix empty keys

frenck commented:
@tjorim, I really like the effort you are putting off here.
Nevertheless, the PR becomes really big. Almost too big to handle.

Please use smaller PR's., those are easier to move forward.

tjorim committed:
Update Configuration Variables sections

tjorim commented:
@frenck You're probably right, I'm sorry. Is it ok if I stop now?

tjorim committed:
Fix Liquid error

tjorim committed:
Merge remote-tracking branch 'upstream/current' into patch-1

tjorim commented:
New PR: https://github.com/home-assistant/home-assistant.io/pull/5929
